### PR TITLE
Android restful implementation

### PR DESCRIPTION
**Pull request by kylelwm**

### CODE REVIEW

File: client/android/WorldScope/app/src/main/java/com/litmus/worldscope/FacebookLoginActivity.java [hunk 1]
```diff
@@ -1,27 +1,81 @@
...
+import android.widget.Toast;
 
 import com.facebook.AccessToken;
 
 import layout.FacebookLoginFragment;
+import retrofit2.Call;
+import retrofit2.Callback;
+import retrofit2.Response;
 
 public class FacebookLoginActivity extends FragmentActivity implements FacebookLoginFragment.OnFragmentInteractionListener {
```
Comment (sharminelow) [inline client/android/WorldScope/app/src/main/java/com/litmus/worldscope/FacebookLoginActivity.java]: If we are using the v7 support library, I think using `extends AppCompatActivity` would be a better choice. AppCompatActivity is the direct child class of FragmentActivity of support v4. [stackoverflow link](http://stackoverflow.com/questions/31297246/activity-appcompatactivity-fragmentactivity-and-actionbaractivity-when-to-us)

File: client/android/WorldScope/app/src/main/java/com/litmus/worldscope/MainActivity.java [hunk 2]
```diff
@@ -1,28 +1,52 @@
...
-
+import android.support.design.widget.TabLayout;
 import android.support.v4.app.Fragment;
 import android.support.v4.app.FragmentManager;
 import android.support.v4.app.FragmentPagerAdapter;
 import android.support.v4.view.ViewPager;
-import android.os.Bundle;
-import android.view.LayoutInflater;
+import android.util.Log;
+import android.view.View;
```
Comment (sharminelow) [inline client/android/WorldScope/app/src/main/java/com/litmus/worldscope/MainActivity.java]: The import statements can be arranged. `import android.view.View;` can be moved downloads to join the other `android.view`

**Comment by sharminelow** [comment]
I don't see any other major issues.

File: client/android/WorldScope/app/src/main/java/com/litmus/worldscope/MainActivity.java (hunk 2 above)
Comment (kylelwm) [inline client/android/WorldScope/app/src/main/java/com/litmus/worldscope/MainActivity.java]: Noted, I've refactored the import statements, please take a look again =)

File: client/android/WorldScope/app/src/main/java/com/litmus/worldscope/FacebookLoginActivity.java (hunk 1 above)
Comment (kylelwm) [inline client/android/WorldScope/app/src/main/java/com/litmus/worldscope/FacebookLoginActivity.java]: Understood, changed to utilize AppCompatActivity instead, please take a look.

**Comment by sharminelow** [comment]
Okay. Good to merge

**Comment by kylelwm** [comment]
Thanks =)

**Pull request merged**